_static/files/command-name/command-name.rst

```diff
@@ -0,0 +1,27 @@
+**To list the available widgets**
```

Comment:
We should make sure to remove the `command-name.zip` file now that we have this one.